apps/src/codebridge/Codebridge.tsx

```diff
@@ -42,10 +44,12 @@ export const Codebridge = React.memo(
     startSource,
     onRun,
     onStop,
+    projectVersion,
   }: CodebridgeProps) => {
     const reducerWithCallback = useReducerWithCallback(
       projectReducer,
-      setProject
+      setProject,
+      new Set(PROJECT_REDUCER_ACTIONS.REPLACE_PROJECT)
```

Comment:
We do -not- want to fire off the setProject callback if we're calling replaceProject - that's because that action only fires if a new project comes in externally. Since the external source already has the new project, there's no need to re-notify it.